portfolio/src/main/webapp/talent.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8">
+    <title>Pictures Of Jaden</title>
```

Comment:
If this is a placeholder for the talent page, can you change the page title to identify it as such?

Reply:
Got it.